project/StdBits.scala

```diff
@@ -419,6 +419,93 @@ object StdBits {
     )
   }
 
+  /** Extracts all the native libraries from `sqlite-jdbc-<version>.jar`.
+    * Also, removes the `SqliteJdbcFeature` class from the JAR.
+    */
+  def extractNativeLibsFromSqlite(
```

Comment:
This is complicated... no wonder only few people are able to integrate `Standard` libraries properly.